server/trucks.js

```diff
@@ -253,4 +254,16 @@ var foodTrucks = [
  * that method should take a single parameter that represents the day to filter on
  * use underscore's filter() method to return all trucks that have the day in their
  * schedule
- * /
\ No newline at end of file
+ */
+
+var filterByDay = function filterByDay(day) {
+
+	var todayFoodTrucks = _.filter(foodTrucks, function(truck){
+		for (var i = 0; i < truck.schedule.length; i++){
+			return day === truck.schedule[i];
```

Comment:
You want to check if this is true before returning: `if (day === truck.schedule[i]) { return true; }` By not checking if it's true before returning, the function call returns before looping through all schedule values. It would only return true if day happened to be the first value in the schedule array.